### PR TITLE
chore(input): un-expose Key.BaseRune + Key GoDocs

### PR DESCRIPTION
**Pull request by meowgorithm**

### CODE REVIEW

File: input/key.go [hunk 1]
```diff
@@ -179,31 +179,79 @@ const (
...
+
+	// AltRune is the actual, unshifted key pressed by the user. For example,
+	// if the user presses shift+a, or caps lock is on, the AltRune will be
+	// 'a'.
+	//
+	// In the case of non-latin keyboards, like Arabic, AltRune is the
+	// unshifted key on the keyboard.
+	//
+	// This is only available when enhanced keyboard modes are enabled:
+	// fixterms, the Kitty Keyboard protocol, or the Windows Console API.
```
Comment (aymanbagabas) [inline input/key.go:200]:
```suggestion
	// the Kitty Keyboard protocol, or the Windows Console API.
```

File: input/key.go [hunk 2]
```diff
@@ -179,31 +179,79 @@ const (
...
+
+	// baseRune is the key pressed according to the standard PC-101 key layout.
+	// On internaltional keyboards, this is the key that would be pressed if
+	// the keyboard was set to US layout.
+	//
+	// For example, if the user presses 'q' on a French AZERTY keyboard, the
+	// baseRune will be 'q'.
+	//
+	// This is only available when enhanced keyboard modes are enabled:
+	// fixterms, the Kitty Keyboard protocol, or the Windows Console API.
```
Comment (aymanbagabas) [inline input/key.go:211]:
```suggestion
	// the Kitty Keyboard protocol, or the Windows Console API.
```

File: input/key.go [hunk 3]
```diff
@@ -179,31 +179,79 @@ const (
...
+	baseRune rune
+
+	// IsRepeat indicates whether the key is being held down and sending events
+	// repeatedly.
+	//
+	// This is only available when enhanced keyboard modes are enabled:
+	// fixterms, the Kitty Keyboard protocol, or the Windows Console API.
```
Comment (aymanbagabas) [inline input/key.go:218]:
```suggestion
	// the Kitty Keyboard protocol, or the Windows Console API.
```

**Review by aymanbagabas — COMMENTED** [review]
Love this, some nitpicks

**Comment by meowgorithm** [comment]
Cool, going to accept these and then adjust a tiny bit more.

**Comment by meowgorithm** [comment]
Okay cool, I think we're good here if you do, @aymanbagabas.

**Pull request merged**